src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Backup/BackupOptionsType.php

```diff
@@ -26,22 +26,71 @@
 
 namespace PrestaShopBundle\Form\Admin\Configure\AdvancedParameters\Backup;
 
+use PrestaShop\PrestaShop\Adapter\Configuration;
 use PrestaShopBundle\Form\Admin\Type\SwitchType;
-use Symfony\Component\Form\AbstractType;
+use PrestaShopBundle\Form\Admin\Type\TranslatorAwareType;
 use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Translation\TranslatorInterface;
 
 /**
  * Class BackupOptionsType builds form for backup options.
  */
-class BackupOptionsType extends AbstractType
+class BackupOptionsType extends TranslatorAwareType
 {
+    /**
+     * @var Configuration
+     */
+    private $configuration;
+
+    /**
+     * BackupOptionsType constructor.
+     *
+     * @param TranslatorInterface $translator
+     * @param array $locales
+     * @param Configuration $configuration
+     */
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        Configuration $configuration
+    ) {
+        parent::__construct($translator, $locales);
+        $this->configuration = $configuration;
+    }
+
     /**
      * {@inheritdoc}
      */
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
+        $backupAllHelp = $this->trans(
+            'Drop existing tables during import.',
+            'Admin.Advparameters.Help'
+        );
+
+        $backupAllHelp .= '<br>';
+        $backupAllHelp .= str_replace(
+            '%prefix%',
+            $this->configuration->get('_DB_PREFIX_'),
+            '%prefix%connections, %prefix%connections_page %prefix%connections_source, %prefix%guest, %prefix%statssearch'
+        );
+
+        $backupDropTablesHelp = $this->trans(
+            'If enabled, the backup script will drop your tables prior to restoring data.',
+            'Admin.Advparameters.Help'
+        );
+
+        $backupDropTablesHelp .= ' ';
+        $backupDropTablesHelp .= $this->trans('(ie. "DROP TABLE IF EXISTS")', 'Admin.Advparameters.Help');
```

Comment:
Why not doing a single string?

Reply:
Well, good question! I thought it was because the example might be different in some cases but if it is not the case, you are right, we could definitely use one string.

Reply:
@JevgenijVisockij Please could you delete this string, please?

Reply:
Removed the string.